jbrowse/src/org/labkey/jbrowse/JBrowseController.java

```diff
@@ -752,26 +771,11 @@ public Object execute(GetSessionForm form, BindException errors) throws Exceptio
             }
             else
             {
-                resp = new JSONObject();
-                resp.put("session", form.getSession());
+                throw new FileNotFoundException("File not found.");
```

Comment:
small thing, but it's not really a file we normally want, right?

Reply:
Right now this is covering the case in which a user provides a session we don't have a file for, which currently is any session that isn't demo. Should this toss a different exception or return an error object?

Reply:
I'll grant this is all a placeholder, but two things:

1) I would make the message 'Unknown session' or something. It's a little more appropriate for the situation
2) more importantly, rather than throw, follow the pattern that happened in validateForm(). I'm typing here so it might have syntax errors, but something like:

```
errors.reject(ERR_MSG, "Unknown session: " + form.getSession());
return null;

```

Reply:
Just updated in the commit "errors.reject instead of throwing FileNotFoundException"